packages/kbn-es/src/cluster.js

```diff
@@ -237,9 +238,12 @@ exports.Cluster = class Cluster {
    * @param {Object} options
    * @property {string|Array} options.esArgs
    * @property {string} options.esJavaOpts
+   * @property {Boolean} options.skipNativeRealmSetup
    * @return {undefined}
    */
-  _exec(installPath, options = {}) {
+  _exec(installPath, opts = {}) {
+    const { skipNativeRealmSetup = false, ...options } = opts;
```

Comment:
This doesn't feel super elegant, but I also was trying to avoid restructuring a bunch of items in `Cluster` (feels higher risk as there's no TS and few tests). 

Open to other ideas. The main issue is that we always try to set up the native realm on `start`, but this doesn't work for a multi-node cluster with multiple master nodes, as the cluster hasn't been formed until all masters are online.

Reply:
> but this doesn't work for a multi-node cluster with multiple master nodes,

What is the problem to run a real-world scenario with a single master and one or two `data` nodes?

Reply:
Sorry, I should have said with multiple _master-eligible_ nodes. I was trying to replicate a real-world scenario here by including each node in the `initial_master_nodes`, since we recommend each cluster [have at least 3 of them](https://www.elastic.co/guide/en/elasticsearch/reference/current/high-availability-cluster-design.html#high-availability-cluster-design). These aren't dedicated masters.

Since master-eligible nodes will elect the first master, the cluster isn't fully formed until a quorum of them is online, which is why the native realm setup fails.

Also, I was trying to make this as flexible as possible so it could support a variety of node configurations, since folks writing these tests might not know the finer details (like "our integration tests support multi-node clusters as long as you are only configuring a single master).